contracts/Synthswap.sol

```diff
@@ -265,7 +261,7 @@ contract SynthSwap is ISynthSwap, Owned, ReentrancyGuard {
                 IERC20(desc.srcToken).safeTransferFrom(msg.sender, address(this), desc.amount);
             }
             // approve AggregationRouterV4 to spend srcToken
-            IERC20(desc.srcToken).safeApprove(address(router), desc.amount);
+            IERC20(desc.srcToken).approve(address(router), desc.amount);
```

Comment:
@JaredBorders just a heads up I ended up just falling back onto `approve` just because it had the functionality we needed and was better on gas than checking allowance and decreasing. But let me know if you have any concerns